satellite/satellitedb/overlaycache.go

```diff
@@ -636,6 +636,7 @@ func (cache *overlaycache) UpdateAddress(ctx context.Context, info *pb.Node, def
 			dbx.Node_TotalAuditCount(0),
 			dbx.Node_UptimeSuccessCount(0),
 			dbx.Node_TotalUptimeCount(0),
+			dbx.Node_PieceCount(0),
```

Comment:
make sure to add the field to `overlaycache.Get` as well

Reply:
:+1: